src/Raven.Server/Dashboard/DatabasesInfoRetriever.cs

```diff
@@ -315,13 +315,13 @@ private static DatabaseOngoingTasksInfoItem GetOngoingTasksInfoItem(DocumentData
             return new DatabaseOngoingTasksInfoItem()
             {
                 Database = database.Name,
-                ExternalReplicationCount = extRepCountOnNode,
-                ReplicationHubCount = replicationHubCountOnNode,
-                ReplicationSinkCount = replicationSinkCountOnNode,
+                ReplicationCount = extRepCountOnNode,
```

Comment:
why we changed that?

Reply:
but this is hidden knowledge and no one will remenber about that.

Reply:
Revert those changes, export this type if needed and use keyof.

Reply:
done